### PR TITLE
Community health index

### PR DESCRIPTION
**Pull request by wtrocki**
## Motivation

Add standard github documents and templates. 

Each file is pushed as separate commit and it can be reviewed or rejected.
This change will help to tick all the boxes in the community profile.
screen shot 2017-11-29 at 9 40 51 am

It's worth having this +more than 20 github stars to raise project profile. 

## Suggestions

Additionally developers can add CODEOWNERS for assigning reviewers for specific parts of the code.

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+* Follow the coding style in use.
+* Check for unnecessary whitespace with `git diff --check` before committing.
+* Make sure you have added the necessary tests for your changes.
+* Run _all_ the tests to assure nothing else was accidentally broken.
+
+## Submitting changes
+
+* Push your changes to a topic branch in your fork of the repository.
+* Submit a pull request to the repository in the [RainCatcher GitHub organization]
+  (https://github.com/feedhenry-raincatcher) and choose branch you want to patch
```
Comment (maleck13) [inline .github/CONTRIBUTING.md]: remove raincatcher ref

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  * Make sure you fill in the earliest version that you know has the issue.
+* Fork the repository on GitHub.
+
+## Making changes
+
+* Create a topic branch from where you want to base your work.
+  * This is usually the master branch.
+  * To quickly create a topic branch based on master; `git checkout -b
+    <branch name> master`. By convention we typically include the JIRA issue
+    key in the branch name, e.g. `RAINCATCH-1234-my-feature`.
```
Comment (maleck13) [inline .github/CONTRIBUTING.md]: Prob want to change the project from RAINCATCH

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+keep this process as easy as possible so we've outlined a few guidelines below.
+For more information about the FeedHenry project, visit
+[our website](http://feedhenry.org/).
+
+## Asking for help
+
+Whether you're contributing a new feature or bug fix, or simply submitting a
+ticket, the RainCatcher team is available for technical advice or feedback.
+You can reach us at [gitter chat](https://gitter.im/FeedhenryRaincatcher/Lobby) or the
+[feedhenry-raincatcher list](https://www.redhat.com/mailman/listinfo/feedhenry-raincatcher)
```
Comment (maleck13) [inline .github/CONTRIBUTING.md]: rain catcher ref

**Comment by wtrocki** [comment]
@maleck13 It looks like you got some outdated version when reviewing. RainCatcher was already removed.

**Pull request merged**